src/misc.test.ts

```diff
@@ -109,10 +109,7 @@ describe('miscellaneous', () => {
         [
           [{}, 'a'],
           [{ a: 1 }, 'b'],
-          // Object.hasOwnProperty does not work for arrays
-          // [['foo'], 0],
-          // [['foo'], '0'],
-        ] as any[]
+        ] as const
```

Comment:
These array test cases didn't seem relevant because they contradicted the input type.